netdata.spec.in

```diff
@@ -1,90 +1,100 @@
+#
+# Conditional build:
+%bcond_without	systemd		# systemd
+%bcond_without	nfacct		# build with nfacct plugin
+
 %if 0%{?fedora} || 0%{?rhel} >= 7
-%global do_systemd 1
 %else
-%global do_systemd 0
+%undefine	with_systemd
 %endif
 
 Summary:	Linux real time system monitoring, over the web
 Name:		@PACKAGE_NAME@
 Version:	@PACKAGE_RPM_VERSION@
 Release:	@PACKAGE_RPM_RELEASE@%{?release_suffix}%{?dist}
-License:	GPLv2+
-URL:		http://firehol.org
-Source:		%{name}-@PACKAGE_VERSION@.tar.bz2
-
+License:	GPL v3+
+Group:		Applications/System
+Source0:	https://github.com/firehol/netdata/archive/v%{version}/%{name}-%{version}.tar.gz
+URL:		http://netdata.firehol.org/
+BuildRequires:	autoconf
+BuildRequires:	automake
 BuildRequires:	libmnl-devel
+%{?with_nfacct:BuildRequires:	libnetfilter_acct-devel}
+BuildRequires:	pkgconfig
 BuildRequires:	zlib-devel
-Requires:	libmnl
-Requires:	zlib
-%if !0%{?rhel}
-BuildRequires:	libnetfilter_acct-devel
-Requires:	libnetfilter_acct
-%endif
-%if 0%{do_systemd}
-BuildRequires:		systemd
-Requires(post):		systemd
+%if %{with systemd}
+BuildRequires:	systemd
+Requires(post):	systemd
```

Comment:
we have automated tool ([adapter](https://github.com/pld-linux/rpm-build-tools/blob/master/adapter.awk)) that reformats .spec for common style. and it assumes ts=4, not ts=8. restored.